login.html

```diff
@@ -47,10 +47,56 @@
                   <input type="checkbox" value="remember-me">
                   Remember Password
                 </label>
+
+            <script type="text/JavaScript">
+              function validate(form)  
+                     {
+                        var returnValue=true;
+                        if(frmLogin.txtUserName.value.not entered)
```

Comment:
What is frmLogin? Where did you get that?

Reply:
`txtUserName.value.not entered` how is this working? Have you tried this thing on your local machine if it is running or not?